docs/guidance/map.md

```diff
@@ -1,8 +1,15 @@
 # Map
 
+````{ifconfig} language and language not in ('es') or not language
 <div class="video-wrapper">
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/3Q_fQBsju6Y" title="Matching your data to OCDS fields" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
+````
+````{ifconfig} language and language == 'es'
+<div class="video-wrapper">
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/wL0wiTaEpW4" title="Mapeo de sus datos con los campos OCDS" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+````
```

Comment:
We only need 3 backticks, since we aren't nesting backticked sections.

Also, I think the conditions can be simplified.

Similar changes can be made to the other locations.

````suggestion
```{ifconfig} language != 'es'
<div class="video-wrapper">
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/3Q_fQBsju6Y" title="Matching your data to OCDS fields" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
```
```{ifconfig} language == 'es'
<div class="video-wrapper">
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/wL0wiTaEpW4" title="Mapeo de sus datos con los campos OCDS" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
```
````

Reply:
Ah true, I think I tried that way and for some reason, it didn't work, but it is working now so,  I changed it now